packages/perspective/webpack-plugin.js

```diff
@@ -7,84 +7,61 @@
  *
  */
 
-const WORKER_LOADER_PATH = require.resolve("./src/loader/file_worker_loader");
-const WASM_LOADER_PATH = require.resolve("./src/loader/cross_origin_file_loader.js");
-const BLOB_LOADER_PATH = require.resolve("./src/loader/blob_worker_loader.js");
+const FILE_WORKER_LOADER = require.resolve("./loader/file_worker_loader");
+const CROSS_ORIGIN_FILE_LOADER = require.resolve("./loader/cross_origin_file_loader.js");
+const BLOB_LOADER = require.resolve("./loader/blob_worker_loader.js");
 
-const BABEL_CONFIG = require("./babel.config.js");
+const include = [__dirname];
 
 class PerspectiveWebpackPlugin {
     constructor(options = {}) {
         this.options = options;
     }
 
     apply(compiler) {
```

Comment:
This refactored plugin does not work - it fails to build the [webpack example project in `examples/`](https://github.com/jpmorganchase/perspective/tree/master/examples/webpack).

Reply:
Fixed in latest comment.